domains/jax.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Jackson's Portfolio",
+  "repo": "https://github.com/jxaq/jxaq.github.io",
+  "owner": {
+    "username": "jxaq",
+    "email": "wild.dives.0x@icloud.com"
+  },
+  "record": {
+    "TXT": "v=spf1 -all",
+    "A": "76.76.21.21"
```

Comment:
A records should be within an array. Same goes for the TXT records.

Reply:
Ok, should be fixed now.